pkg/plugin/trans.go

```diff
@@ -313,7 +313,9 @@ func (r resourceTransitioner[T]) handleRequestedGeneric(
 		// the factor.
 		maxIncrease := (remainingReservable / opts.factor) * opts.factor
 		// ... but we must allow at least opts.forceApprovalMinimum
-		maxIncrease = util.Max(maxIncrease, opts.forceApprovalMinimum)
+		remainingBelowForcedApproval := util.SaturatingSub(opts.forceApprovalMinimum, r.pod.Reserved)
+		increaseFromForceApproval := (remainingBelowForcedApproval / opts.factor) * opts.factor
```

Comment:
Why do we need to align it with the `factor` here? What will happen if we don't?

Reply:
Fair question — after thinking about it some more, I think it's not needed. In theory we can hit that if a VM is created with bounds that aren't a multiple of the configured compute units, but there isn't _really_ a correct way to handle that.

Removed it in a204813627eb0b4d9da828c5931b1e6c0ea3c413